### ISSUE TITLE
init.d script not working

### ISSUE THREAD

**Issue by chr1s-t0pher** [stargieg/bacnet-feed#1]
on latest openwrt and latest RUTOS
init script not working, says cant see files in /tmp/.uci/, 
files are in /etc/config/

manually starting bacserv-uci works

Thu Oct 31 13:30:30 2024 daemon.err bacserv-uci[23577]: /tmp/.uci/bacnet_dev is missing or corrupt
Thu Oct 31 13:30:30 2024 daemon.err bacserv-uci[23577]: Failed to load config file bacnet_dev
Thu Oct 31 13:30:30 2024 daemon.err bacserv-uci[23577]: /tmp/.uci/bacnet_ai is missing or corrupt
Thu Oct 31 13:30:30 2024 daemon.err bacserv-uci[23577]: Failed to load config file bacnet_ai

**Comment by stargieg** [comment]
I remember seeing and fixing the same error. I'll take another look at it. Thanks for the hint.

**Comment by stargieg** [comment]
Workaround: Type on the console
```
sed -i /etc/init.d/bacserv -e 's/daemon/root/'
```
The Idea was to let bacserv running as user daemon and not as root.
```
procd_set_param user daemon
```

**Comment by stargieg** [comment]
issue source d3cf925074654d489c8398b57bbfc5ebf87b8a5e
issue fix cfd7e23acdee85993792d0551d19979cc5c4c6e3
